packages/components/src/context-actions/ContextMenu.tsx

```diff
@@ -594,7 +594,7 @@ class ContextMenu extends PureComponent<ContextMenuProps, ContextMenuState> {
     if (pendingItems.length > 0) {
       pendingElement = (
         <div className="loading">
-          <LoadingSpinner />
+          <LoadingSpinner className="mimic-fa-layers-vertical-align" />
```

Comment:
Not crazy about needing to provide a long class name in many of these cases... should we have a `verticalAlign` prop or something that maps to a class name instead? Or any other way to do this?

Reply:
I was thinking eventually we should use flex positioning for all of the cases and gradually get rid of the "hack" class. It's kind of a flag that this is styled in a deprecated way. That said, I'm not a big fan of it either, was just trying to not muck with too much of the existing layout. I'm fine to change this to a prop instead.

Reply:
Or just a different class name; like we have `loading-spinner-large`, maybe `loading-spinner-vertical-align`